portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase

```diff
@@ -193,14 +193,35 @@ definition {
 	}
 
 	@description = "Verify that a Form's Submit button's text can be customized"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test SubmitButtonTextCanBeCustomized {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-131028 SubmitButtonTextCanBeCustomized pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.addTextField(fieldPositionNumber = "0");
+
+		FormsAdminNavigator.gotoSettings();
+
+		Form.configureSubmitLabel(submitLabel = "Send Form");
+
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		Form.viewSubmitButtonText(key_buttonText = "Send Form");
 
+		FormPortlet.submitLocalizedSuccessfully(buttonName = "Send Form");
```

Comment:
Hey @leilanyaragao, the last step planned for this test is `Assert the Success Page`, but this line is just asserting that a success message is being displayed. In light of this, can you please assert that the Success Page is also being displayed?

The macro `viewSuccessPage` will do it for you. :slightly_smiling_face: 

Sorry for not tell it to you before, but this information just came to my mind only a few minutes after finish the review of this pull. :grin: